src/sidebar/components/TopBar.tsx

```diff
@@ -91,7 +100,7 @@ function TopBar({
         className={classnames(
           'container flex items-center h-full',
           // Text sizing will size icons in buttons correctly
-          'text-xl'
+          'text-[16px]'
```

Comment:
`text-xl` and `text-[16px]` are interchangeable here, but I felt that removing one layer of abstraction was useful. We're trying to make the icons 16px square, so hiding that behind a named text size that one has to go look up felt unhelpful.